alpha/external-web/app/eligibility-client.js

```diff
@@ -47,6 +47,25 @@ module.exports = {
         callback(error, null)
       }
     })
+  },
+
+  /**
+   * Update an existing claimant.
```

Comment:
This copy pasted comment is an example of why I don't like comments as standard.